OPHD/Things/Structures/Structure.h

```diff
@@ -110,6 +110,7 @@ class Structure: public Thing
 	int turnsToBuild() const { return mTurnsToBuild; }
 	int age() const { return mAge; }
 	int maxAge() const { return mMaxAge; }
+	bool ages() const { return maxAge() > 0; }
```

Comment:
I couldn't think of a better name. Oftentimes good to include is or has prefix for function names that return a bool.

Reply:
Indeed. Can't think of a suggestion for this one.